src/components/Sum.jsx

```diff
@@ -1 +1,11 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+)
+
+Sum.propTypes = {
+  a: PropTypes.number.isRequired,
```

Comment:
it's not a required
![image](https://user-images.githubusercontent.com/91411692/231131706-121daf30-dc1b-45f1-a383-c323741a104b.png)